src/Cake.DocFx.Tests/Build/DocFxBuildRunnerTests.cs

```diff
@@ -1,4 +1,6 @@
-﻿namespace Cake.DocFx.Tests.Build
+﻿using Cake.Core.IO;
```

Comment:
usings should be within namespace to match the style of the other files in the project